### PR TITLE
Mark for implicit directory modification date change.

### PR DESCRIPTION
**Pull request by dkocher**

### CODE REVIEW

File: ftp/src/main/java/ch/cyberduck/core/ftp/FTPProtocol.java [hunk 1]
```diff
@@ -36,6 +36,11 @@ public String getPrefix() {
         return String.format("%s.%s", FTPProtocol.class.getPackage().getName(), StringUtils.upperCase(this.getType().name()));
     }
 
+    @Override
+    public DirectoryTimestamp getDirectoryTimestamp() {
+        return DirectoryTimestamp.implicit;
```
Comment (AliveDevil) [inline ftp/src/main/java/ch/cyberduck/core/ftp/FTPProtocol.java:41]: is it though?
I could see Microsoft FTP not adhering to this, or at least Microsoft FTP on NTFS drives, where modification dates are not propagated up the tree for directories.

Comment (dkocher) [reply ftp/src/main/java/ch/cyberduck/core/ftp/FTPProtocol.java:41]: Yes, this also applies to UNIX. Modification dates are only updated for the parent folder and this assumption is not correct.

Comment (dkocher) [reply ftp/src/main/java/ch/cyberduck/core/ftp/FTPProtocol.java:41]: Currently only Nextcloud/ownCloud have ETag and timestamps [propagate to the root](https://github.com/owncloud/client/wiki/Etags-and-file-ids#etag-change). Other WebDAV implementations *may* also support this.

Comment (AliveDevil) [reply ftp/src/main/java/ch/cyberduck/core/ftp/FTPProtocol.java:41]: SFTP on Windows won't be implicit as well.

Comment (AliveDevil) [reply ftp/src/main/java/ch/cyberduck/core/ftp/FTPProtocol.java:41]: So Windows would be direct parent folder only, not propagating up.

Comment (dkocher) [reply ftp/src/main/java/ch/cyberduck/core/ftp/FTPProtocol.java:41]: Clarified in afff5eba that the definition only applies to direct parent folder.

**Review by ylangisc — APPROVED** [review]

**Pull request merged**